frontend/src/features/election_management/components/CheckElectionDefinition.tsx

```diff
@@ -0,0 +1,84 @@
+import { useState } from "react";
+
+import { Alert } from "@/components/ui/Alert/Alert";
+import { Button } from "@/components/ui/Button/Button";
+import { InputField } from "@/components/ui/InputField/InputField";
+import { t, tx } from "@/lib/i18n";
+import { Election, RedactedEmlHash } from "@/types/generated/openapi";
+import { formatDateFull } from "@/utils/format";
+
+import { RedactedHash, Stub } from "./RedactedHash";
+
+interface CheckElectionDefinitionProps {
+  file: File;
+  election: Election;
+  hash: RedactedEmlHash;
+}
+
+export function CheckElectionDefinition({ file, election, hash }: CheckElectionDefinitionProps) {
+  const [stubs, setStubs] = useState<Stub[]>(
+    hash.redacted_indexes.map((redacted_index) => ({
+      selected: false,
+      index: redacted_index,
+    })),
+  );
+
+  return (
+    <section className="md">
+      <h2>{t("election.check_eml.title")}</h2>
+      <p>
+        {tx("election.check_eml.description", {
+          file: () => {
+            return <strong>{file.name}</strong>;
+          },
+        })}
+      </p>
+      <Alert type="notify" variant="no-icon" margin="mb-md" small>
+        <p>
+          <strong>{election.category}</strong>
+          <br />
+          <span className="capitalize">{formatDateFull(new Date(election.election_date))}</span>
+        </p>
+        <div>
+          <span>
+            <strong>{t("digital_signature")}</strong>({t("hashcode")}):
```

Comment:
Missing space, page says: "Digitale vingerafdruk(hashcode):"

```suggestion
            <strong>{t("digital_signature")}</strong> ({t("hashcode")}):
```